budget-app/src/app/services/create_budget.service.ts

```diff
@@ -2,15 +2,27 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Budget } from '../classes/budget';
+import { Category } from '../classes/category';
 
 @Injectable({
   providedIn: 'root'
 })
 export class BudgetCreateService {
-  serverUrl: string = 'http://localhost:8080/budgetCreate'
+  serverUrl: string = 'http://localhost:8080//budgetCreateController/budgetCreate'
+  serverUrlCat: string = 'http://localhost:8080//budgetCreateController/CatCreate'
```

Comment:
Why are there two forward slashes after the port?
EDIT: I see it now. In BudgetCreateConroller.java, if you remove the forward slash from the @RequestMapping you won't need to use two forward slashes here